.npmrc

```diff
@@ -0,0 +1,9 @@
+# pnpm configuration
+# Equivalent to .yarnrc.yml settings
+
+# Store location (local, not global)
+store-dir=~/.pnpm-store
```

Comment:
The comment says "local, not global" but the setting is global, relative to $HOME. Should we use the default and omit the explicit setting?

Reply:
would just need to update the ci caching to dynamically fetch the path with `pnpm store path` but that's ez enough to do